platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -118,4 +155,4 @@ class ViewerRetrieveStudyData extends Component {
   }
 }
 
-export default ViewerRetrieveStudyData;
+export default withSnackbar(ViewerRetrieveStudyData);
```

Comment:
This is a pretty simple component. We could just make it a functional component?

Reply:
yes.. sure.. can we cover on a separate PR?